.travis.yml

```diff
@@ -1,10 +1,15 @@
 sudo: required
 
+scala:
+  - 2.12.10
+  - 2.13.1
```

Comment:
I'm not sure this works, you need to do the sbt ++ trick.
See https://github.com/sksamuel/avro4s/blob/master/.travis.yml

Reply:
oh I knew I forgot something. Thanks!